Installation/CHANGES.md

```diff
@@ -70,7 +70,10 @@ Release date: March 2019
 -   Added the class `CGAL::Rigid_triangle_mesh_collision_detection` to
     detect intersections between meshes and volumes undergoing affine
     transformations.
-
+    
```

Comment:
This will have to be moved to the 5.0 section.

Reply:
I've marked it as a todo.